composer.json

```diff
@@ -5,17 +5,14 @@
 	"type": "wordpress-plugin",
 	"license": "GPL-3.0-or-later",
 	"require": {
-		"php": "^7.0",
+		"php": "^7.1",
 		"composer/installers": "~1.2"
 	},
 	"require-dev": {
-		"phpunit/phpunit": "^6.0",
+		"phpunit/phpunit": "9.5.14",
 		"woocommerce/woocommerce-sniffs": "0.1.0",
 		"dave-liddament/sarb": "^1.1",
-		"yoast/phpunit-polyfills": "^1.0",
-		"phpdocumentor/reflection-docblock": "4.3.4",
-		"phpdocumentor/type-resolver": "0.5.1",
-		"doctrine/instantiator": "1.0.5"
```

Comment:
Not sure why these were here. I'm guessing it was to help lock in the versions required for the older version of phpunit. Subscriptions Core doesn't require these but they are dependencies of phpunit.